scripts/install/config.default.sh

```diff
@@ -0,0 +1,40 @@
+# These are the default configs
+# DON'T EDIT THIS FILE
+# If you want to use different values, create a file named 'config.sh'
+# If present, 'config.sh' will be used as well in addition to 'config.default.sh' (to override variables).
+
+# Url of the hybris backend
+# Will replace default host (https://localhost:9002) as a backend endpoint
+# Make sure you specify the full url for the backend (https://[host]:[port]
+BACKEND_URL="https://localhost:9002"
+
+BASE_SITE="electronics-spa"
+CURRENCY="usd"
+OCC_PREFIX="/occ/v2/"
+
+SPARTACUS_PROJECTS=(
+        "core"
+        "assets"
+        "storefrontlib"
+        "storefrontstyles"
+        "cds"
```

Comment:
I think we can skip CDS here now and create ticket to allow app instalation with CDS (with dev schematics)

Reply:
OK